pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -203,6 +193,29 @@ def _get_format_data(self, fmt):
         data.attrs = self._prune_attributes(data.attrs)
         return data
 
+    def get_format_type_data(self, format_name):
+        """Get the format type data.
+
+        For example: ext4
+
+        :param format_name: a name of the format type
+        :return: an instance of DeviceFormatData
+        """
+        fmt = get_format(format_name)
+        return self._get_format_type_data(fmt)
+
+    def _get_format_type_data(self, fmt):
+        """Get the format type data.
+
+        :param fmt: an instance of DeviceFormat
+        :return: an instance of DeviceFormatData
+        """
+        data = DeviceFormatData()
+        data.type = fmt._type or ""
+        data.mountable = fmt.mountable
```

Comment:
@vojtechtrefny, I cannot use the format class, because I still need the instance to check if the format is mountable.